src/pom.xml

```diff
@@ -694,6 +704,12 @@
                 <groupId>org.acegisecurity</groupId>
```

Comment:
Acegi is need for jasypt 1.8 whom encoder implements its interface. Removing that jar is not a straight forward thing since it is required to modifies the geostore password encoder: jasypt 1.9 seems to be not backward compatible